app/views/services/govbox/register_step1.html.erb

```diff
@@ -1,24 +1,28 @@
-<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
-
-<%= javascript_include_tag ENV.fetch('AUTOFORM_JAVASCRIPT_URL') %>
-
 <div class="container">
-  <%= form_tag register_step2_services_govbox_index_path, data: {'slovensko-digital-autoform': ENV.fetch('AUTOFORM_ACCESS_TOKEN')} do %>
+  <%= form_tag register_step2_services_govbox_index_path do %>
+
     <div class="row">
       <div class="col-lg-8">
-        <h1 class="form-title">Registrácia k službe GovBox</h1>
-        <p class="form-subtitle lead">Zadajte údaje právnickej osoby, pre ktorú máte záujem zriadiť službu.</p>
-
-        <div class="row">
-          <div class="col-md-8 form-group">
-            <%= label_tag :legal_subject_name, 'Názov právnickej osoby' %>
-            <%= text_field_tag :legal_subject_name, params[:legal_subject_name], class: 'form-control input-lg', required: true, autocomplete: 'off', data: {'slovensko-digital-autoform': 'name'} %>
-            <span class="form-text text-muted">Napríklad: Slovensko.Digital, Martinus</span>
-          </div>
-          <div class="col-md-4 form-group">
-            <%= label_tag :cin, 'IČO' %>
-            <%= text_field_tag :cin, params[:cin], class: 'form-control input-lg', required: true, data: {'slovensko-digital-autoform': 'cin'} %>
-            <span class="form-text text-muted">Napríklad: 36421928</span>
+        <h1 class="form-title">Výber služby</h1>
+        <div class="row mt-4">
+          <div class="col-lg-12">
+            <div class="row">
+              <div class="col-md-12">
+                <div class="form-check">
+                  <label class="form-check-label">
+                    <%= radio_button :mode, params[:mode], 'sync_mode', class: 'form-check-input', required: true %>
+                    <span>Mám záujem o <strong>GovBox</strong> na preposielanie správ zo schránky na email.</span>
+                  </label>
+                </div>
+                <div class="form-check">
+                  <label class="form-check-label">
+                    <%= radio_button :mode, params[:mode], 'api_mode', class: 'form-check-input', required: true %>
+                    <span>Mám záujem o <strong>GovBox API</strong> na integráciu vlastného systému.</span>
```

Comment:
```suggestion
                    <span>Mám záujem o <strong>GovBox API</strong> na integráciu vlastného informačného systému.</span>
```

Reply:
@pavolzbell nebude to uz potom takmer doslova replikacia poznamky pod tym?
![image](https://user-images.githubusercontent.com/44023416/89906813-25226f80-dbec-11ea-97cc-3ee551b2c3ba.png)

Reply:
@smith558 podla mna nie a dal to slovo uz do vyberu, ta poznamka nie je az taka dolezita

Reply:
Ta poznamka je divna. Pomocou ineho informacneho systemu znamena co?

Reply:
Cela tato formulacia je divna "prostrednictvom API alebo pomocou ineho informacneho systemu", ved aj ten IS musi ist cez API predsa

Reply:
dajme ju takto:

GovBox API nepreposiela správy zo schránky na email ale umožňuje manipuláciu so schránkou prostredníctvom programového rozhrania (API). Ak neviete, čo to znamená, neváhajte nás <kontaktovať>. (link na kontaktny mail)

Reply:
@pavolzbell Okay